cli/js/lib.deno.ns.d.ts

```diff
@@ -2236,17 +2236,20 @@ declare namespace Deno {
         signal?: number;
       };
 
-  export interface RunOptions {
+  export interface RunStdioOptions {
+    stdout?: ProcessStdio | number;
+    stderr?: ProcessStdio | number;
+    stdin?: ProcessStdio | number;
+  }
+
+  export interface RunOptions extends RunStdioOptions {
```

Comment:
Is there a benefit in breaking RunOptions into two parts? It seems you could do the same trick with the original RunOptions.

Reply:
The problem was that `RunOptions` has non-optional fields, so where I currently have
```ts
let fileServer: Deno.Process<{ stdout: "piped" }>;
```
it would have to be
```ts
let fileServer: Deno.Process<Deno.RunOptions & { stdout: "piped" }>;
```

Reply:
I see... Doesn't this seem like more complexity than it's worth?

Reply:
Reverted to the previous commit which just uses `RunOptions`.